workers/Dockerfile.tests

```diff
@@ -0,0 +1,23 @@
+FROM continuumio/miniconda3:4.3.11
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+COPY workers/environment.yml .
+
+RUN conda env create -f environment.yml
+
+ENV PATH "/opt/conda/envs/workers/bin:$PATH"
+
+COPY data_models/dist/data-refinery-models-* data_models/
```

Comment:
The answer to this question is similar to the answer to the one above it. Because the [RUN command](https://docs.docker.com/engine/reference/builder/#run) is always executed within the context of the container I cannot run `(ls data_models -1 | tail -1)` in the Dockerfile and have it interact with files/directories within the host machine without first copying the files into the container. If it were possible, I certainly would agree that figuring out the correct file and THEN copying it into the container would be greatly preferable.